lib/apiservers/portlayer/swagger.json

```diff
@@ -59,6 +59,115 @@
 				}
 			}
 		},
+		"/kv": {
```

Comment:
please - if it can be done quickly. Otherwise we'll have an issue to address it before the next tag.